flow_test/oisst_recipe.py

```diff
@@ -116,7 +116,7 @@ def register_recipe(recipe: BaseRecipe):
         "https://www.ncei.noaa.gov/data/sea-surface-temperature-optimum-interpolation"
         "/v2.1/access/avhrr/{yyyymm}/oisst-avhrr-v02r01.{yyyymmdd}.nc"
     )
-    dates = pd.date_range("2019-09-01", "2021-01-05", freq="D")
+    dates = pd.date_range("2010-12-25", "2021-01-05", freq="D")
```

Comment:
@tracetechnical Can you revert this change?  This will result in a very large recipe test.

Reply:
Sure!

Reply:
Fixed in later commit.